config.omero

```diff
@@ -0,0 +1,12 @@
+# By default, we enable 'minimal_webapp' only
+config append omero.web.apps '"minimal_webapp"'
+# Comment this out to enable
```

Comment:
"Uncomment this"?

Reply:
In the instructions I have at https://github.com/ome/ome-documentation/pull/2015 ask users to use this config when installing and running the minimal_webapp in Docker. At that point they won't have installed the react_webapp so adding it to the omero.web.apps would cause an ImportError.

Reply:
But shouldn't the comment say, "uncomment this out to enable" ?